connectors/sources/mysql.py

```diff
@@ -331,37 +331,24 @@ async def fetch_documents(self, database, table, query):
                 f"Skipping {table} table from database {database} since no primary key is associated with it. Assign primary key to the table to index it in the next sync interval."
             )
 
-    async def _fetch_all_databases(self):
-        """Fetches all user databases
-
-        Returns:
-            List: List of databases
-        """
-
-        # Query to get all databases
-        query = QUERIES["ALL_DATABASE"]
-        _, query_response = await self._execute_query(query=query)
-        databases = [database[0] for database in query_response]
-        return databases
-
     async def get_docs(self):
         """Executes the logic to fetch databases, tables and rows in async manner.
 
         Yields:
             dictionary: Row dictionary containing meta-data of the row.
         """
         database_config = self.configuration["database"]
-        if isinstance(database_config, str):
+        if not database_config:
```

Comment:
let's be explicit.

`if  database_config == ''`

Reply:
Sure we will update that in the other Validate PR.